packages/orbit-components/src/Card/CardSection/CardSection.ct.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import { CardSectionInput } from "./CardSection.ct-story";
+
+test.describe("interaction", () => {
+  test("typing spaces works on non-expandable sections", async ({ mount }) => {
+    const component = await mount(<CardSectionInput />);
+    const input = component.getByTestId("input");
+
+    await input.focus();
```

Comment:
isn't this should be normally tested with testing-library?

Reply:
we don't have `CardSection` Jest tests, so I created this. and it has one big benefit — it tests the actual browser environment, not a simulated one

Reply:
hm, can we add it to a jest then? it's a bit strange to have it under visual regression 🤔 And don't think it matters here what environment it is. Sorry for complications, I just feel it's not the purpose of visual regression to test cases like this 🙂

Reply:
if we want to agree to use Playwright strictly for visual tests, then yes, I can move em, for consistency. though I personally see value in _any_ component tests being in it

Reply:
Yep, let's use it for visual testing only for now. We have testing-library and I do not think we will get rid of it in near feature in favor of playwright, having it tested half here and half there -> isn't a good idea probably. I like playwright experimentals, it seems a better solution than chromatic :) Until now I thought we have it as a replacement for chromatic, not for the whole testing 🙂

Reply:
Resolved. Jest is not failing on this case, playwright does. We can have it in playwright.